aerosonicdb/models/mfcc/mlp.py

```diff
@@ -101,42 +130,49 @@ def run_cv(train_path=TRAIN_PATH,
         ap_score = average_precision_score(y_true=y_test, y_score=y_prob)
         eval_results.append(ap_score)
 
-        PrecisionRecallDisplay.from_predictions(y_test, y_prob, ax=ax, name=f'MLP {count}')
+        PrecisionRecallDisplay.from_predictions(
+            y_test, y_prob, ax=ax, name=f"MLP {count}"
+        )
 
         if save_models:
             # save the model
-            model_path = os.path.join(output_path, f'mlp_{count}', 'model')
+            model_path = os.path.join(output_path, f"mlp_{count}")
 
             if not os.path.exists(model_path):
                 os.makedirs(model_path)
 
-            est.model_.save(model_path)
+            est.model_.save(os.path.join(model_path, "model.keras"))

```

Comment:
keras seems to crash if trying to save filename without a `.keras` extension nowadays. updated all relevant places here and in CNN model as well.